packages/flutter/test/widgets/overscroll_stretch_indicator_test.dart

```diff
@@ -821,23 +875,29 @@ void main() {
     await tester.pumpAndSettle();
     expect(find.text('Index 1'), findsOneWidget);
     double lastStretchedLocation = tester.getCenter(find.text('Index 1')).dy;
-    expect(lastStretchedLocation, greaterThan(51.0));
+    if (!isImpeller) {
```

Comment:
For all of these tests where `if (!impeller)` was added, we should definitely be checking for when it is impeller.